### DISCUSSION TITLE
Moodle and ant-design

### DISCUSSION THREAD

**Discussion by Camber62** [ant-design/ant-design#49945] [General]
When using ant, the user, in response to moodle, issues a link:documentSlice.ts:17 Warning: [antd: notification] A static function cannot use context like a dynamic theme. Please use the "Application" component instead.
Although everything is normal
 <ConfigProvider
        locale={ruRu}
        theme={themeConfig}
    >
        <App className='app-ai_course_management'>
            <Provider store={store}>

                <Main/>
            </Provider>
        </App>
    </ConfigProvider>

**Comment by Wxh16144** [comment]
https://ant.design/docs/blog/why-not-static explains very well why there are error warning.

**Reply by Wxh16144** [reply]
You may also need to pay attention to the nesting order. see: https://ant.design/components/app#sequence-with-configprovider
